src/main/kotlin/no/nav/syfo/service/SykmeldingService.kt

```diff
@@ -62,12 +63,16 @@ class SykmeldingService constructor(
         kafkaValidationResultProducer: KafkaProducer<String, ValidationResult>,
         kafkaManuelTaskProducer: KafkaProducer<String, ProduceTask>,
         sm2013ManualHandlingTopic: String,
-        sm2013BehandlingsUtfallTopic: String
-
+        sm2013BehandlingsUtfallTopic: String,
+        kafkaproducerPapirSmRegistering: KafkaProducer<String, PapirSmRegistering>,
+        sm2013SmregistreringTopic: String
     ) {
         log.info("Mottatt norsk papirsykmelding, {}", fields(loggingMeta))
         PAPIRSM_MOTTATT_NORGE.inc()
 
+        var ocrFil : Skanningmetadata? = null
```

Comment:
Er det noen spesiell grunn til at denne har blitt var? Det er litt code smell når man må bruke !! lenger ned 😅

Reply:
Ups! Pinlig! Det er jeg helt enig i. Det hang igjen fra noen eksperimenter jeg gjorde, men jeg tror ikke det trengs lenger :)

Reply:
Ah, nei. Det er fordi den hentes ut innenfor try-blokken, men trengs i catch-blokken, hvor vi prøver å gjøre sånn best-effort mapping av felter fra OCR-filen.

Alternativt kan vi wrappe det kalle alene i en egen try-catch, men det er kanskje også litt hårete?

Reply:
Skjønner, da må det kanskje bli sånn ja.. Men for sykmelder kan vi kanskje ta bort !! eller..? :)

Reply:
Nei, det blir dessverre samme problem der også.